source/server/backtrace.cc

```diff
@@ -0,0 +1,14 @@
+#include "server/backtrace.h"
+
+#include <iostream>
+
+namespace Envoy {
+
+bool BackwardsTrace::log_to_stderr_ = false;
+
+void BackwardsTrace::setLogToStderr(bool log_to_stderr) {
+  std::cerr << "setting log_to_stderr_=" << log_to_stderr << std::endl;
```

Comment:
nit: prepend with BackwardsTrace or something?

Reply:
will do; or I could just delete it.

Reply:
Sure either way